### ISSUE TITLE
Scaling of X,Y,Z not written to ept.json

### ISSUE THREAD

**Issue by wonder-sk** [hobu/untwine#7]
In EPT decoder we read offset+scale from ept.json, but `bu` does not write scaling information to the ept.json:
```
	{"name": "X", "type": "floating", "size": 8 },
	{"name": "Y", "type": "floating", "size": 8 },
	{"name": "Z", "type": "floating", "size": 8 },
```

When I edited the ept.json to include scaling, the decoder/renderer started to work:
```
	{"name": "X", "type": "float", "size": 8, "scale": 0.01 },
	{"name": "Y", "type": "float", "size": 8, "scale": 0.01 },
	{"name": "Z", "type": "float", "size": 8, "scale": 0.01 },
```

What is the precedence of scaling/offset information: shall we trust whatever is written in the LAS/LAZ file (and ignore ept.json) or prefer what is stored in ept.json?

A related question is whether ept.json should report X/Y/Z as doubles or as ints (given that's how they are actually stored in LAZ)

**Comment by abellgithub** [comment]
@hobu? @connormanning?

I was under the impression that scale/offset only applied to dimensions other than X/Y/Z.

**Comment by connormanning** [comment]
Scale/offset applies to XYZ as well, otherwise binary/zstandard/other future generic encodings would not work.  LAZ, although the default due to its compression ratio, is kind of the odd one out in that it is re-specified in the encoding itself.

So for LAZ data, the schema should show the scale/offset and advertise their values as int32 values.

**Comment by abellgithub** [comment]
Fixed by 3fbfc9fbfc24c1cdf9aa7629642ef0a294fd7123
